mobile/App.tsx

```diff
@@ -1,20 +1,79 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Image } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { HomeScreen } from './src/screens/HomeScreen';
+import { Settings } from './src/screens/Settings';
+import { ChatScreen } from './src/screens/ChatScreen';
+
+const Tab = createBottomTabNavigator();
 
 const App: React.FC = () => {
   return (
-    <View style={styles.container}>
-      <Text>TODO</Text>
-    </View>
+    <NavigationContainer>
+      <Tab.Navigator
+        initialRouteName="Home"
+        screenOptions={({ route }) => ({
+          tabBarIcon: ({ focused }) => {
+            let icon;
+            const tint = focused ? '#E89228' : '#B8B8B8';
+            let width;
+            let height;
+
+            if (route.name === 'Home') {
+              icon = require('./assets/images/Home.png');
+              width = 29;
+              height = 23.5;
+            } else if (route.name === 'Settings') {
+              icon = require('./assets/images/Cog.png');
+              width = 26;
+              height = 26;
+            } else if (route.name === 'Chat') {
+              icon = require('./assets/images/Chat_bubble.png');
+              width = 22;
+              height = 22;
+            }
+
+            return <Image style={{ width: width, height: height, tintColor: tint }} source={icon} />;
```

Comment:
Also this is kinda messy. How about a map that you can index into with `route.name`? like:

```ts
const TabIcons = {
  Home: <Image style={styles.homeTabIcon} source={require('./assets/images/Home.png')} />,
  ...
};
```

```ts
screenOptions = {({ route }) => ({
  tabBarIcon: () => TabIcons[route.name]
)}
```